### PR TITLE
Body UI wrapper

### PR DESCRIPTION
**Pull request by bhushankhope**

### CODE REVIEW

File: libs/ccf-body-ui/src/lib/body-ui/body-ui.component.ts [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+  /** Rotation for the deck gl */
+  readonly rotation = input(undefined, { transform: numberAttribute });
+  /** Rotation X for the deck gl */
+  readonly rotationX = input(undefined, { transform: numberAttribute });
+  /** Zoom for the deck gl */
+  readonly zoom = input(undefined, { transform: numberAttribute });
+  /** Target for the deck gl */
+  readonly target = input(undefined, { transform: parseTargetInput });
+  /** Bounds for the deck gl */
+  readonly bounds = input<XYZTriplet>();
```
Comment (axdanbol) [inline libs/ccf-body-ui/src/lib/body-ui/body-ui.component.ts:104]: Needs a transform

**Review by axdanbol — CHANGES_REQUESTED** [review]
A couple of small tweaks

**Review by axdanbol — APPROVED** [review]

**Review by axdanbol — CHANGES_REQUESTED** [review]
@bhushankhope The body-ui does not display in the preview. Please investigate

**Comment by bhushankhope** [comment]
> @bhushankhope The body-ui does not display in the preview. Please investigate

Fixed the configuration. Preview is working now

File: apps/body-ui/project.json [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+              "maximumError": "1mb"
+            },
+            {
+              "type": "anyComponentStyle",
+              "maximumWarning": "2kb",
+              "maximumError": "4kb"
+            }
+          ],
+          "outputHashing": "all"
+        },
```
Comment (axdanbol) [inline apps/body-ui/project.json:44]: Please add staging target

File: apps/body-ui/project.json [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            "glob": "**/*",
+            "input": "apps/body-ui/public"
+          }
+        ],
+        "styles": ["apps/body-ui/src/styles.scss"],
+        "scripts": []
+      },
+      "configurations": {
+        "production": {
+          "budgets": [
```
Comment (axdanbol) [inline apps/body-ui/project.json:31]: Add baseHref for production

**Review by axdanbol — CHANGES_REQUESTED** [review]
Some more config updates

**Review by axdanbol — DISMISSED** [review]

**Pull request merged**